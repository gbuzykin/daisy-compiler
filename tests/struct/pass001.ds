namespace N1 {

struct A {
  i : i32;
  f : double;
};

}
