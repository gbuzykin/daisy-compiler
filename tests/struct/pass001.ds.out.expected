./struct/pass001.ds:4:3: debug: defining field `i` of type `i32`
 4 |   i : i32;
   |   ^
./struct/pass001.ds:5:3: debug: defining field `f` of type `f64`
 5 |   f : double;
   |   ^
./struct/pass001.ds: info: warnings 0, errors 0
